src/Features/Core/Portable/EditAndContinue/AbstractEditAndContinueAnalyzer.cs

```diff
@@ -3650,11 +3668,11 @@ private static bool IsExtensionMethodThisParameter(IParameterSymbol parameter)
 
             ReportCustomAttributeRudeEdits(diagnostics, oldSymbol, newSymbol, newNode, newCompilation, capabilities, out var hasAttributeChange, out var hasReturnTypeAttributeChange, cancellationToken);
 
-            ReportUpdatedSymbolDeclarationRudeEdits(diagnostics, oldSymbol, newSymbol, newNode, newCompilation, out var hasGeneratedAttributeChange, out var hasGeneratedReturnTypeAttributeChange, cancellationToken);
+            ReportUpdatedSymbolDeclarationRudeEdits(diagnostics, oldSymbol, newSymbol, newNode, newCompilation, out var hasGeneratedAttributeChange, out var hasGeneratedReturnTypeAttributeChange, out var hasParameterRename, cancellationToken);
             hasAttributeChange |= hasGeneratedAttributeChange;
             hasReturnTypeAttributeChange |= hasGeneratedReturnTypeAttributeChange;
 
-            if (hasAttributeChange || hasReturnTypeAttributeChange)
+            if (hasAttributeChange || hasReturnTypeAttributeChange || hasParameterRename)
```

Comment:
>hasParameterRename [](http://example.com/codeflow?start=70&length=18)

This is odd as parameter names have nothing to do with custom attributes. It is correct as we want to emit metadata update for the parameter name, but perhaps we should rename `AddCustomAttributeSemanticEdits`?

Reply:
Yes, fair point. I'll restructure this a bit. Calling `AddCustomAttributeSemanticEdits` is a bit overkill here